operator/src/test/java/oracle/kubernetes/operator/helpers/KubernetesTestSupport.java

```diff
@@ -164,6 +166,7 @@ public Memento install() {
     supportNamespaced(PVC, V1PersistentVolumeClaim.class, this::createPvcList);
     supportNamespaced(SECRET, V1Secret.class, this::createSecretList);
     supportNamespaced(SERVICE, V1Service.class, this::createServiceList);
+    supportNamespaced(SCALE, Scale.class, null);
```

Comment:
Just a suggestion and not really necessary but you could also use two arg method 'supportNamespaced(SCALE, Scale.class)?